docs-kits/kits/Behaviour Twin MDP Kit/Success Stories/mdp_application.md

```diff
@@ -0,0 +1,74 @@
+---
+id: mdp_application
+title: MDP Application
+description: Behaviour Twin MDP Application Success Story
+---
+
+![Model Based Development and Data Processing Kit banner](@site/static/img/kit-icons/behaviour-mdp-kit-icon.svg)
+
+### Model Based Development and Data Processing Kit
+
+## Introduction to the Use Case
+
+The Behaviour Twin MDP (Model based development and Data Processing) Application was developed within the framework of the Catena-X initiative, to enable collaborative utilization of behavioural digital twins in the automotive industry. This innovative application is poised to define the workflow and the tool chain for the model-based system development and data processing, leveraging Catena-X technology and collaborative principles. The mission of the Behaviour Twin MDP Application is to provide a comprehensive framework for the selection, integration, and utilization of behavioural digital twins. Through adherence to Catena-X standards and principles, we strive to ensure secure data exchange, interoperability, and compatibility, laying the foundation for seamless collaboration and efficiency in innovation.
+
+The objective is to enhance model-based design and data processing using models coming from component suppliers (for example: the OEM gets the models of the gear box, the brakes and the battery from the respective Tier1s), by facilitating model exchange between Tier1s and OEM or between Tier2s and Tier1. This is done using the industry standard FMU (which is a black box model, so the intellectual property is protected) and performing the exchange over EDC. The MDP KIT defines the functionality of the software applications needed for the MDP use cases, that is to combine the models into a simulation architecture and perform parametric studies, but also to do field data processing based on evaluation models of the component suppliers.
+
+## General Information Display: BehBehaviouravior Twin MDP Application
+
+The General Information Display of the Behaviour Twin MDP (Model and Data Processing) Application offers users a user-friendly interface to visualize essential information about behavioural digital twins and their components. Whether users are analyzing the behaviour of a component within a vehicle or maintaining a system, this feature provides easy access to valuable insights.
+
+A technical expert from the OEM has the task to build a design of a new car. The simulation expert will collect the necessary models from the Tier1 and combine them in a vehicle model to evaluate different designs. The models are exchanged between Catena-X partners using EDC and using the FMU format (black box model).
+
+![Overview Application](../images/Pic_AppStory1.png)
```

Comment:
1. Do you have the confirmation that you are allowed to use the Siemens Logo?
2. In addition the app screenshots look like powerpoint - is this a real application?
3. How is the MDP app differentiated from the Siemens apps? Can you please explain this in the introduction.

Reply:
1. Yes, this has been clarified.
2. This is a real App, based on the Mendix-Tool Suite. Of course, it contains on some of the home screens images taken from Powerpoint (the smaller rectangles on the pages).
3. The MDP app is a proof-of-concept app developed within the Catena-X project to illustrate the MDP work flow and a possible implementation of a user guidance. This is not a commercial Siemens product. This is mentioned now in the introduction section of the MDP app success story.

Reply:
Thanks for the clarification!